README.md

```diff
@@ -55,6 +55,30 @@ And just run:
 jsonschema-gentype
 ```
 
+# Validate
+
+It also provide some validate improvements from `jsonschema` to validate a YAML file.
```

Comment:
This package also provide some validations features for YAML file based on `jsonschema`.